examples/01_plotting/plot_surf_atlas.py

```diff
@@ -0,0 +1,59 @@
+"""
+Loading and plotting of a cortical surface atlas
+=================================================
+
+The Destrieux parcellation (Destrieux et al, 2010) in fsaverage5 space as
+distributed with Freesurfer is used as the chosen atlas.
+
+The :func:`nilearn.plotting.plot_surf_roi` function is used
+to plot the parcellation on the pial surface.
+
+See :ref:`plotting` for more details.
+
+References
+----------
+
+Destrieux et al, (2010). Automatic parcellation of human cortical gyri and
+sulci using standard anatomical nomenclature. NeuroImage, 53, 1.
+URL http://dx.doi.org/10.1016/j.neuroimage.2010.06.010.
+"""
+
+###############################################################################
+from nilearn import plotting
+from nilearn import datasets
```

Comment:
I think that it would be nice to delay these imports to the cell in which they are used first.